samples/maps/demo/geoheatmap-orthographic/demo.js

```diff
@@ -0,0 +1,84 @@
+(async () => {
+    const topology = await fetch(
+            'https://code.highcharts.com/mapdata/custom/world.topo.json'
+        ).then(response => response.json()),
+        colors = Highcharts.getOptions().colors;
+
+    Highcharts.getJSON('https://cdn.jsdelivr.net/gh/highcharts/highcharts@5c536debb0/samples/data/geoheatmap-cities-dataset.json', function (data) {
+        Highcharts.mapChart('container', {
+            chart: {
+                map: topology
+            },
+
+            title: {
+                text: 'GeoHeatMap Series Demo',
+                align: 'left',
+                style: {
+                    textOutline: '2px white'
```

Comment:
Is it necessary? And if so, use `2px contrast` so that it works better with themes.